lib/version.rb

```diff
@@ -0,0 +1,3 @@
+class SidekiqMonitoringBase
+  VERSION = "1.3.3"
+end
```

Comment:
Why add this file? it's not a very standard naming it seems

Reply:
I see from your commit name it's probably because it was blocking the `bundle install`, but in that case if we shall duplicate the the version let's put it in the gemspec directly?

Reply:
Done.